packages/react-components/react-list-preview/stories/List/MultipleActionsDifferentPrimary.stories.tsx

```diff
@@ -0,0 +1,189 @@
+import {
+  Button,
+  Caption1,
+  makeResetStyles,
+  makeStyles,
+  Menu,
+  MenuItem,
+  MenuList,
+  MenuPopover,
+  MenuTrigger,
+  mergeClasses,
+  shorthands,
+  Text,
+  tokens,
+} from '@fluentui/react-components';
+import { List, ListItem, ListProps } from '@fluentui/react-list-preview';
+import { MoreHorizontal20Regular } from '@fluentui/react-icons';
+
+import * as React from 'react';
+
+const useListItemRootStyles = makeResetStyles({
+  position: 'relative',
+  flexGrow: '1',
+  gap: '8px',
+  border: '1px solid grey',
+  alignItems: 'center',
+  borderRadius: '8px',
+  gridTemplate: `"preview preview preview" auto
+      "header action secondary_action" auto / 1fr auto auto
+    `,
+});
+
+const useStyles = makeStyles({
+  list: {
+    display: 'flex',
+    flexDirection: 'column',
+    ...shorthands.gap('16px'),
+    maxWidth: '300px',
+  },
+  listItem: {
+    display: 'grid',
+    ...shorthands.padding('8px'),
+  },
+  caption: {
+    color: tokens.colorNeutralForeground3,
+  },
+  image: {
+    maxWidth: '100%',
+    ...shorthands.borderRadius('5px'),
+  },
+  checkmark: {
+    position: 'absolute',
+    left: '10px',
+    top: '10px',
+    zIndex: 1,
+  },
+});
+
+const CardExample = (props: { title: string; value: string; selected?: boolean }) => {
+  const listItemStyles = useListItemRootStyles();
+  const styles = useStyles();
+  const { value } = props;
+
+  return (
+    <ListItem
+      value={props.value}
+      className={mergeClasses(listItemStyles, styles.listItem)}
+      checkmark={{ className: styles.checkmark }}
+      aria-label={value}
+      onClick={e => {
+        // Prevent the default behavior - toggling the selection
+        e.preventDefault();
+        alert('Primary action triggered!');
+      }}
+    >
+      <div role="gridcell" style={{ gridArea: 'preview', overflow: 'hidden' }}>
+        <img className={styles.image} src={`https://picsum.photos/seed/${value}/300/130/`} alt="Presentation Preview" />
+      </div>
+      <div role="gridcell" style={{ gridArea: 'header' }}>
```

Comment:
Let's use className overrides since we don't want to promote unnecessary inline style usage